gitbootcamp.rst

```diff
@@ -203,6 +203,33 @@ from the `core-workflow  <https://github.com/python/core-workflow>`_
 repository to backport the commit.
 
 
+.. _git_from_mercurial:
+
+Applying a Patch from Mercurial to Git
+--------------------------------------
+
+Scenario:
+
+- A Mercurial patch exists but there is no pull request for it.
+
+Solution:
+
+1. Download the patch locally.
+
+2. Create a new branch and switch to it.
+
+3. For Mac and Unix::
+
+   $ patch -p1 < /path/to/issueNNNN.patch
```

Comment:
Looking at the git apply documentation, there is an option to check the patch first.
```
--check
Instead of applying the patch, see if the patch is applicable to the current working tree and/or the index file and detects errors. Turns off "apply".
```
Do you think it would be worthwhile to add that first and then proceed with the suggestion from @zware if the patch won't apply?